go/classes/page.go

```diff
@@ -0,0 +1,16 @@
+package classes
+
+// PageHeader is the default class used for the PageHeader component
+const PageHeader = "popui-admin-page__header"
+
+// PageHeaderActions is the default class used for the actions slot of PageHeader component
+const PageHeaderActions = "popui-admin-page__header-actions"
+
+// PageContent is the default class used for the PageContent component
+const PageContent = "popui-admin-page__content"
+
+// PageSection is the default class used for the actions slot of PageSection component
+const PageSection = "popui-admin-page__section"
+
+// PageSectionTitle is the default class used for the title of PageSectionTitle component
+const PageSectionTitle = "popui-admin-page__section-title"
```

Comment:
This is a bit overkill, I'd even suggest combining all the constants here into a single file. I'm also not sure they add enough functionality to warrant a package.

My suggestion approach:

```
// Classes used by the page components.
const (
  PageHeader = "popui-admin-page__header"
  PageHeaderActions = "popui-admin-page__header-actions"
  PageContent = "popui-admin-page__content"
  PageSection = "popui-admin-page__section"
  PageSectionTitle = "popui-admin-page__section-title"
)
```